v20.1/build-a-spring-app-with-cockroachdb-jdbc.md

```diff
@@ -591,6 +628,76 @@ The annotations listed at the top of the `JdbcApplication` class definition decl
+
+When the application is started, all of the queries specified by the changesets are executed in the order specified by their `changeset` tag's `id` value. At application startup, Liquibase also creates a table called [`databasechangelog`](https://docs.liquibase.com/concepts/databasechangelog-table.html) in the database where it performs changes. This table's rows log all completed changesets.
+
+To see the completed changsets, open a new terminal, start the [built-in SQL shell](cockroach-sql.html), and query the `databasechangelog` table:
+
+<section class="filter-content" markdown="1" data-scope="secure">
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --certs-dir=certs
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="insecure">
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure
+~~~
+
+</section>
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM roach_data.databasechangelog;
+~~~
+
+~~~
+  id | author |             filename              |           dateexecuted           | orderexecuted | exectype |               md5sum               |                                              description                                               | comments | tag  | liquibase | contexts | labels | deployment_id
+-----+--------+-----------------------------------+----------------------------------+---------------+----------+------------------------------------+--------------------------------------------------------------------------------------------------------+----------+------+-----------+----------+--------+----------------
+  1  | root   | classpath:db/changelog-master.xml | 2020-06-17 14:57:01.431506+00:00 |             1 | EXECUTED | 8:939a1a8c47676119a94d0173802d207e | sqlFile                                                                                                |          | NULL | 3.8.9     | crdb     | NULL   | 2420221402
+  2  | root   | classpath:db/changelog-master.xml | 2020-06-17 14:57:01.470847+00:00 |             2 | EXECUTED | 8:c2945f2a445cf60b4b203e1a91d14a89 | insert tableName=account; insert tableName=account; insert tableName=account; insert tableName=account |          | NULL | 3.8.9     | crdb     | NULL   | 2420221402
+(2 rows)
+~~~
+
+Typically, Liquibase properties are defined in a separate [`liquibase.properties`](https://docs.liquibase.com/workflows/liquibase-community/creating-config-properties.html) file. In this application, `application.yml`, the properties file that sets the [Spring properties](https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html), also includes some properties that enable and configure Liquibase:
+
+~~~ yml
+...
+ liquibase:
+   change-log: classpath:db/changelog-master.xml
+   default-schema:
+   drop-first: false
+   contexts: crdb
```

Comment:
Worth mentioning perhpas that spring profiles can be used to run different Liquibase changesets for testing or different environments. In that case its only a matter of linking the same name of the spring profiles with the Liquibase 'contexts'. There's only one profile+context in this application for simplicity.